### DISCUSSION TITLE
remove emailVerified field

### DISCUSSION THREAD

**Discussion by httpdennis** [nextauthjs/next-auth#4397] [Help]
how can i remove the "emailVerified" field from my application? i tried a lot (with prismaAdapter and own adapter), but i didn't found a way to remove it. I am using Prisma Database.

**Comment by iCodePoet** [comment]
I have the same issue 😭

**Comment by Zerquix18** [comment]
Same. I don't have this field in my User schema, but Prisma says it can't recognize it

**Comment by Erxtc** [comment]
😭😭

**Comment by negu63** [comment]
I referred to this solution. -> [Override prisma adapter](https://github.com/nextauthjs/next-auth/discussions/2197#discussioncomment-6135911)

This is my prisma `User` model.
```prisma
model User {
    id       String    @id @default(cuid())
    name     String?
    nickname String?
    email    String?   @unique
    image    String?
    mobile   String?
}
```


Just return an object excluding `emailVerified`.
```typescript
const prismaAdapter = PrismaAdapter(prisma);

// @ts-ignore
prismaAdapter.createUser = (data: User) => {
  return prisma.user.create({
    data: {
      email: data.email,
      image: data.image,
      mobile: data.mobile,
      nickname: data.nickname,
      name: data.name,
    },
  });
};
```

And pass `prismaAdapter` to the `adapter` option.
```typescript
export const authOptions: NextAuthOptions = {
    ...
    adapter: prismaAdapter,
    ...
}
```

I'm not sure why you have to do this.😅
